.buildkite/dagster-buildkite/dagster_buildkite/steps/integration.py

```diff
@@ -165,6 +167,29 @@ def build_auto_materialize_perf_suite_steps():
     )
 
 
+def skip_if_not_azure_commit():
+    """If no dagster-azure files are changed, skip the azure live tests."""
+    return (
+        None
+        if (any("dagster-azure" in str(path) for path in ChangedFiles.all))
+        else "Not a dagster-azure commit"
```

Comment:
nit: I super-like the optimization, but what if another package breaks it? It's also the goal of integration tests to catch these. Ensuring this is run occasionally even if there is no commit might be preferable

Reply:
See change to oss pipeline: we still run these every day on the nightly build.